lib/taxjar/api/request.rb

```diff
@@ -7,6 +7,10 @@ class Request
       DEFAULT_API_URL = 'https://api.taxjar.com'
       SANDBOX_API_URL = 'https://api.sandbox.taxjar.com'
 
+      class << self
+        attr_accessor :logger
```

Comment:
Having the logger on class level allows configuring it globally for all requests.

Reply:
@cema-sp This LGTM if you move the configuration as @prsimp suggested. 👍